py-scripts/lf_create_vap_cv.py

```diff
@@ -19,21 +19,41 @@
             # Load an existing Chamber View scenario
```

Comment:
Was a bit confused by the commit messages, as they're both the same, but I see the body contains the actual change.